src/main/webapp/WEB-INF/static/js/trialmanager/germplasm-study-source/germplasmStudySource.js

```diff
@@ -322,9 +322,13 @@
 					{
 						targets: "germplasm-link-column",
 						render: function (data, type, rowData, meta) {
-							return '<a class="gid-link" href="javascript: void(0)"'
-								+ ` onclick="openGermplasmDetailsPopup('${rowData.gid}')">`
-								+ EscapeHTML.escape(data) + '</a>';
+							if ($scope.hasAnyAuthority(PERMISSIONS.VIEW_GERMPLASM_DETAILS)) {
```

Comment:
Move $scope.hasAnyAuthority(PERMISSIONS.VIEW_GERMPLASM_DETAILS) outside of render so that it is only executed once.